e2e/multi-svc-app/copilot/front-end/manifest.yml

```diff
@@ -44,4 +44,6 @@ storage:
 variables:
   TEST_JSON_ENV_VAR: "{}" # https://github.com/aws/copilot-cli/issues/1292
   TEST_BOOL_ENV_VAR: on # https://github.com/aws/copilot-cli/issues/1120
-  TEST_JOB_CHECK_VAR: "no"
\ No newline at end of file
+  TEST_JOB_CHECK_VAR: "no"
+
+env_file: magic.env
```

Comment:
ohh what's the behavior if both `variables` and `env_file` is specified at the same time and they contain the same variable name but with a different value?

Reply:
ECS will honor whatever they have in `variables` and execute the env vars specified in the `env_file` in order (see [here](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/taskdef-envfiles.html#taskdef-envfiles-considerations)), which is not the magic by Copilot so I didn't put it in your e2e test.